### DISCUSSION TITLE
MenuItem support of download attribute

### DISCUSSION THREAD

**Discussion by jsheppard-twilio** [twilio-labs/paste#2077] [Ideas] [answered]
Hi,

Currently the MenuItems doesn't support the download attribute.
As it support a href attribute and renders an a tag, I would have guessed we could have used the download attribute.
https://paste.twilio.design/components/menu/#submenubutton-and-menuitem

Is there plans to add this in future?

Our current use case is to allow users to download an invoice by format:
<img width="189" alt="Screenshot 2021-11-29 at 11 19 31" src="https://user-images.githubusercontent.com/59604283/143859035-0ea0acb0-0b97-4f0f-8b5a-29f76b1fef04.png">

**Comment by kylegilbreath** [comment]
Hi @jsheppard-twilio! Thanks for the question/request! I've just created a [backlog ticket](https://issues.corp.twilio.com/browse/DSYS-3151) to address this, and will follow up on this discussion when we've added this functionality to Paste. 

In the meantime, you could check out the [Data Export pattern](https://paste.twilio.design/patterns/data-export/) we have detailed in the docs site that should work for the use case you described, but I did not realize we don't already the download attribute support for those MenuItems to properly support this pattern.

**Comment by loreina** [comment, marked as answer]
Marking this as answered due to inactivity.
